util/src/main/java/org/datacommons/proto/Debug.java

```diff
@@ -0,0 +1,4416 @@
+// Generated by the protocol buffer compiler.  DO NOT EDIT!
```

Comment:
Should generated files get checked in as well?

Reply:
Its a TODO there to remove them soon, but they are checked-in now until the protoc workflow is figured out right... so that presubmit works and IDE doesn't show errors until protoc is run (Mixer I believe checks them in too).